.cirrus.yml

```diff
@@ -25,6 +25,7 @@ task:
   setup_script: ./dev/bots/cirrus_setup.sh
   matrix:
     - name: docs
+      skip: "!changesInclude('packages/flutter/**', 'packages/flutter_test/**', 'packages/flutter_drive/**', 'packages/flutter_localizations/**', 'packages/flutter_goldens/**', 'bin/internal/**') && $CIRRUS_BRANCH != 'master'"
```

Comment:
I think we still want these on presubmit - they don't upload on presubmit, just validate that the docs can be generated.  /cc @gspencergoog

Reply:
Talked offline - I was misunderstanding this, it actually means we always run on master.